docs/intro/contributing.txt

```diff
@@ -117,38 +117,41 @@ Download the Django source code repository using the following command:
 
 Now that you have a local copy of Django, you can install it just like you would
 install any package using ``pip``. The most convenient way to do so is by using
-a *virtual environment* (or virtualenv) which is a feature built into Python
+a *virtual environment* which is a feature built into Python
 that allows you to keep a separate directory of installed packages for each of
 your projects so that they don't interfere with each other.
 
-It's a good idea to keep all your virtualenvs in one place, for example in
-``.virtualenvs/`` in your home directory. Create it if it doesn't exist yet:
+It's a good idea to keep all your virtual environments in one place, for example
+in ``.virtualenvs/`` in your home directory.
 
-.. console::
-
-    $ mkdir ~/.virtualenvs
-
-Now create a new virtualenv by running:
+Create a new virtual environment by running:
 
 .. console::
 
-    $ python -m venv ~/.virtualenvs/djangodev
+    $ python3 -m venv ~/.virtualenvs/djangodev
 
 The path is where the new environment will be saved on your computer.
 
-.. admonition:: For Ubuntu users
+.. admonition:: For Debian/Ubuntu systems
 
-    On some versions of Ubuntu the above command might fail. Use the
-    ``virtualenv`` package instead, first making sure you have ``pip3``:
+    On Debian/Ubuntu systems the above command will fail. It will tell
+    you what to do, follow those instructions.
 
     .. code-block:: console
 
-        $ sudo apt-get install python3-pip
-        $ # Prefix the next command with sudo if it gives a permission denied error
-        $ pip3 install virtualenv
-        $ virtualenv --python=`which python3` ~/.virtualenvs/djangodev
+        $ python3 -m venv ~/virtualenvs/djangodev
+        The virtual environment was not created successfully because ensurepip is not
+        available.  On Debian/Ubuntu systems, you need to install the python3-venv
+        package using the following command.
+
+            apt-get install python3-venv
+
+        You may need to use sudo with that command.  After installing the python3-venv
+        package, recreate your virtual environment.
+
+        Failing command: ['/root/virtualenvs/djangodev/bin/python3', '-Im', 'ensurepip', '--upgrade', '--default-pip']
```

Comment:
`/home/myuser` instead of `/root` ?

Reply:
what about `['.../virtualenvs/djangodev/bin/python3,'`

Reply:
fair

Reply:
done

Reply:
done